spring-vault-core/src/main/java/org/springframework/vault/core/VaultOperations.java

```diff
@@ -54,6 +54,13 @@
 	 */
 	VaultKeyValueOperations opsForKeyValue(String path, KeyValueBackend apiVersion);
 
+	/**
+	 * Return {@link VaultKeyValueMetadataOperations}
+	 *
+	 * @return the operations interface to interact with the Vault Key/Value metadata backend
+	 */
+	VaultKeyValueMetadataOperations opsForKeyValueMetadata();
```

Comment:
Metadata operations should be obtained from `VaultVersionedKeyValueOperations.opsForMetatada()` or `.metadata()`. This way, we preserve the secret backend path and put the operations interface into an area where it belongs to.